File: data/data.code.typescript.client/AckSceneExit.ts
namespace proto {
export class AckSceneExit
{
	private _uid: number;


	constructor(packet?: game.util.Packet) {
		if (packet) {
			this._uid = packet.ReadUint();
		}
	}


	public get uid(): number { return this._uid; }
	public set uid(value: number) { this._uid = value; }
}
}

File: data/data.code.golang.client/ack_scene_exit.go
package proto

import (
	"packet"
)

type AckSceneExit struct {
	Uid                      uint32
}

func AckSceneExitDecode(pack *packet.Packet) *AckSceneExit {
	ackSceneExit := &AckSceneExit{}

	ackSceneExit.Uid = pack.ReadUint32()
	return ackSceneExit
}
